invenio_access/__init__.py

```diff
@@ -33,6 +33,7 @@
 >>> app.config.update({
 ...     'SQLALCHEMY_DATABASE_URI': db_url,
 ...     'SQLALCHEMY_TRACK_MODIFICATIONS': False,
+...     'SECRET_KEY': 'CHANGE_ME',
```

Comment:
`SECRET_KEY`, because it was `None` and then it failed in `invenio_accounts.init_app` -> `flask_security.core..._get_serializer`.

Tried to traceback if there was a default set before, or the serialization was skipped... didn't find it